packages/block-editor/src/components/block-list/use-block-props/use-event-handlers.js

```diff
@@ -48,100 +45,103 @@ export function useEventHandlers( ref, clientId ) {
+					keyCode !== BACKSPACE &&
+					keyCode !== DELETE
+				) {
 					return;
 				}
 
-				selectBlock( clientId );
-			}
+				if ( target !== node || isTextField( target ) ) {
+					return;
+				}
 
-			ref.current.addEventListener( 'focusin', onFocus );
+				event.preventDefault();
 
-			return () => {
-				ref.current.removeEventListener( 'focusin', onFocus );
-			};
-		}
-
-		/**
-		 * Interprets keydown event intent to remove or insert after block if
-		 * key event occurs on wrapper node. This can occur when the block has
-		 * no text fields of its own, particularly after initial insertion, to
-		 * allow for easy deletion and continuous writing flow to add additional
-		 * content.
-		 *
-		 * @param {KeyboardEvent} event Keydown event.
-		 */
-		function onKeyDown( event ) {
-			const { keyCode, target } = event;
-
-			if (
-				keyCode !== ENTER &&
-				keyCode !== BACKSPACE &&
-				keyCode !== DELETE
-			) {
-				return;
+				if ( keyCode === ENTER ) {
+					insertDefaultBlock( {}, rootClientId, index + 1 );
+				} else {
+					removeBlock( clientId );
+				}
 			}
 
-			if ( target !== ref.current || isTextField( target ) ) {
-				return;
+			function onMouseLeave( { buttons } ) {
+				// The primary button must be pressed to initiate selection.
+				// See https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/buttons
+				if ( buttons === 1 ) {
+					onSelectionStart( clientId );
+				}
 			}
 
-			event.preventDefault();
-
-			if ( keyCode === ENTER ) {
-				insertDefaultBlock( {}, rootClientId, index + 1 );
-			} else {
-				removeBlock( clientId );
+			/**
+			 * Prevents default dragging behavior within a block. To do: we must
+			 * handle this in the future and clean up the drag target.
+			 *
+			 * @param {DragEvent} event Drag event.
+			 */
+			function onDragStart( event ) {
+				event.preventDefault();
 			}
-		}
 
-		function onMouseLeave( { buttons } ) {
-			// The primary button must be pressed to initiate selection.
-			// See https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/buttons
-			if ( buttons === 1 ) {
-				onSelectionStart( clientId );
-			}
-		}
-
-		/**
-		 * Prevents default dragging behavior within a block. To do: we must
-		 * handle this in the future and clean up the drag target.
-		 *
-		 * @param {DragEvent} event Drag event.
-		 */
-		function onDragStart( event ) {
-			event.preventDefault();
-		}
-
-		ref.current.addEventListener( 'keydown', onKeyDown );
-		ref.current.addEventListener( 'mouseleave', onMouseLeave );
-		ref.current.addEventListener( 'dragstart', onDragStart );
-
-		return () => {
-			ref.current.removeEventListener( 'mouseleave', onMouseLeave );
-			ref.current.removeEventListener( 'keydown', onKeyDown );
-			ref.current.removeEventListener( 'dragstart', onDragStart );
-		};
-	}, [
-		isSelected,
-		rootClientId,
-		index,
-		onSelectionStart,
-		insertDefaultBlock,
-		removeBlock,
-		selectBlock,
-	] );
+			node.addEventListener( 'keydown', onKeyDown );
```

Comment:
In this line throws an error when we focus out of the selected block, because of `node === null`. I added a not null check on top of the function and the error goes away. 

I don't know how/if this affects something else and not sure if we should remove previous attached event handlers when node is `null`.

Reply:
Should be fixed now.